.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: ruby
+cache: bundler
+rvm:
+  - 2.4.1
+script:
+  - bundle install
```

Comment:
You don't need to do `bundle install` Travis will do that automatically for every ruby projects